handbook/engineering/search/index.md

```diff
@@ -2,9 +2,93 @@
 
 We own the end-to-end Sourcegraph search experience.
 
+## Vision
+
+Sourcegraph search is:
+
+- **Fast:** Performance of showing results is fast and I can get to what I’m looking for quickly.
+- **Expressive:** It is possible to construct a search query to find exactly the results I’m looking for. The tools, syntax, filters, are provided to construct the queries I want to write.
+The answer to “Can I do ‘X’ with Sourcegraph?” will always be yes.
```

Comment:
Either:

> We _strive_ for a search experience that is...

 to preface

> The answer to “Can I do ‘X’ with Sourcegraph?” will always be yes.

or, 

> Sourcegraph search is...

  followed by

 > The answer to “Can I do ‘X’ with Sourcegraph?” _should_ always be yes.

The current phrasing just feels matter-of-fact (it is so), which seems inaccurate, when I think we are going for something aspirational (we continually strive to make it so) or a desired trait (according to a specification or desired behavior), rather than "it is definitely so".